lonboard/_cli.py

```diff
@@ -0,0 +1,108 @@
+# path = Path("/Users/kyle/Downloads/UScounties.geojson")
+
+
+def read_pyogrio(path: Path) -> pa.Table:
+    """Read path using pyogrio and convert field metadata to geoarrow
+
+    Args:
+        path: Path to file readable by pyogrio
+    """
+    meta, table = read_arrow(path)
+    # TODO: assert there are not two column names of wkb_geometry
+
+    # Rename wkb_geometry to geometry
+    geometry_column_index = [
+        i for (i, name) in enumerate(table.column_names) if name == "wkb_geometry"
+    ][0]
+
+    schema = table.schema
+    field = schema.field(geometry_column_index)
+
+    metadata: Dict[bytes, bytes] = field.metadata
+    if metadata.get(b"ARROW:extension:name") == b"ogc.wkb":
+        # Parse CRS and create PROJJSON
+        ext_meta = {"crs": CRS.from_user_input(meta["crs"]).to_json_dict()}
+
+        # Replace metadata
+        metadata[b"ARROW:extension:name"] = b"geoarrow.wkb"
+        metadata[b"ARROW:extension:metadata"] = json.dumps(ext_meta).encode()
+
+    new_field = field.with_name("geometry").with_metadata(metadata)
+    new_schema = schema.set(geometry_column_index, new_field)
+    return pa.Table.from_arrays(table.columns, schema=new_schema)
+
+
+def read_geoparquet(path: Path):
+    """Read GeoParquet file at path using pyarrow
+
+    Args:
+        path: Path to GeoParquet file
+    """
+    file = pq.ParquetFile(path)
+    geo_meta = file.metadata.metadata.get(b"geo")
+    if not geo_meta:
+        raise ValueError("Expected geo metadata in Parquet file")
+
+    table = file.read()
+
+    geo_meta = json.loads(geo_meta)
+    geometry_column_name = geo_meta["primary_column"]
+    geometry_column_index = [
+        i for (i, name) in enumerate(table.schema.names) if name == geometry_column_name
+    ][0]
+
+    crs_dict = geo_meta["columns"][geometry_column_name]["crs"]
+
+    # Parse CRS and create PROJJSON
+    ext_meta = {"crs": crs_dict}
+
+    metadata = {
+        b"ARROW:extension:name": b"geoarrow.wkb",
+        b"ARROW:extension:metadata": json.dumps(ext_meta).encode(),
+    }
+
+    new_field = table.schema.field(geometry_column_index).with_metadata(metadata)
+    new_schema = table.schema.set(geometry_column_index, new_field)
+    return pa.Table.from_arrays(table.columns, schema=new_schema)
+
+
+@click.command()
+@click.option("-o", "--output", type=click.Path(path_type=Path), default=None)
+@click.argument("files", nargs=-1, type=click.Path(path_type=Path))
+def main(output: Optional[Path], files: List[Path]):
+    """Interactively view geospatial data using kepler.gl"""
+
+    tables = []
+    for path in files:
+        if path.suffix == ".parquet":
+            table = read_geoparquet(path)
+        else:
+            table = read_pyogrio(path)
+
+        tables.append(table)
+
+    map_ = viz(tables)
+
+    # If -o flag passed, write to file; otherwise write to stdout
+    if output:
+        map_.to_html(output)
+    else:
+        map_.to_html(sys.stdout)
```

Comment:
in one project I've done something like 


```python
import tempfile


if not output:
    with tempfile.NamedTemporaryFile(suffix=".html", delete=False) as f:
        output = f.name

map_.to_html(f.name)

click.launch(output)
```

tempfile will automatically create a temporary file in `/tmp` which will get deleted by the system at one point. 

if we go this way we could add a warning printing the path to the file.